### PR TITLE
Fix title and description on social and site

### PR DESCRIPTION
**Pull request by omesser**
~Machine Learning Engineering Management~ - no

See [deployed](https://mlem-ai-fix-desc-title-hbswpll.herokuapp.com/)

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ const apiMiddleware = require('@dvcorg/websites-server/src/middleware/api')
 const redirectsMiddleware = require('@dvcorg/websites-server/src/middleware/redirects')
 
 const siteMetadata = {
-  title: 'MLEM - Machine Learning Engineering Management',
-  description: 'Open-source tool to simplify ML model deployment',
+  title: 'MLEM - Simplifying Machine Learning Model Deployment',
+  description: 'The Open-Source tool to simplify your ML model deployments',
```
Comment (aguschin) [inline gatsby-config.js:9]:
```suggestion
  description: 'Doing `Ops` for you, so you could focus on `ML`',
```

**Review by aguschin — APPROVED** [review]
Better than it was, thanks. Can we come up with something better here? Don't have any bright ideas yet, I have to commit.

File: gatsby-config.js (hunk 1 above)
Comment (aguschin) [reply gatsby-config.js:9]: not sure it will be good for SEO though

Comment (aguschin) [reply gatsby-config.js:9]:
```suggestion
  description: 'Focus on ML part of MLOps and let MLEM do the Ops',
```

Comment (aguschin) [reply gatsby-config.js:9]: maybe this one is better

Comment (omesser) [reply gatsby-config.js:9]: personal thought that bothers me a bit: Using backticks / quotes in the short description looks a bit dirty to me, not sure it's needed, can be:
- Doing the Ops for you, so you could focus on ML
Other suggestion (less abbreviation):
- Seamless model operations and deployments for your ML workflow

Comment (aguschin) [reply gatsby-config.js:9]:
```suggestion
  description: 'Serving models for you, so you could focus on ML',
```

Comment (aguschin) [reply gatsby-config.js:9]: ok, I think this one will work then

Comment (aguschin) [reply gatsby-config.js:9]: Don't know. To be good for seo, it should have all these words, right?

**Comment by omesser** [comment]
@aguschin  - Pushed another change, didn't want "productionize" to be so prominent - not a real word, more of a slang, to niche for site description IMO, also maybe not great for SEO, please TAL

**Review by aguschin — APPROVED** [review]

**Pull request merged**